ci-operator/config/terraform-redhat/terraform-provider-ocm/terraform-redhat-terraform-provider-ocm-main.yaml

```diff
@@ -36,34 +36,32 @@ tests:
     make unit-test
   container:
     from: src
-- as: rosa-classic-e2e-presubmit
-  skip_if_only_changed: ^(LICENSE|OWNERS|README\.md|\.gitignore|\.goreleaser\.yaml)$|^docs/|^subsystem/
+- always_run: false
+  as: rosa-classic-e2e-presubmit
+  skip_if_only_changed: ^(LICENSE|OWNERS|README\.md|\.gitignore|\.goreleaser\.yaml)$|^docs/|^subsystem/|^examples/
```

Comment:
you need to remove the ``skip_if_only_changed`` part I think. only ``always_run: false``